src/sentry/remote_config/docs/api.md

```diff
@@ -114,6 +114,44 @@ Set the DSN's configuration.
 
 ### Delete Configuration [DELETE]
 
-Delete the DSN's configuration.
+Delete the project's configuration.
 
 - Response 204
+
+## Configuration Proxy [/proxy/projects/<project_id>/configuration/]
```

Comment:
@Dav1dde Is this URL name spaced better?  The SDK will call `https://<ingest_host_url>/api/0/proxy/projects/<project_id>/configuration/`.

Reply:
I wouldn't encode any information into the url that this is being proxied. I'd treat the API as if it was directly called from an SDK (which it is, just through a layer of proxies).

Reply:
I'm having trouble name spacing it in a more traditional way because of the lack of org slug.  I could just drop the proxy prefix and make it the singular `project` but it didn't "feel" right.  That said, I can totally do that.  Its just a feeling after all :P

Reply:
My initial instinct (not much thought) would have been something like `/api/:project_id/remote_config/`

Reply:
Got it.  I can rename configuration to remote_config.  The prefix to this url is already `host/api/0` regardless what i define so i think `project/<id>/remote_config/` will have to be the final state of it.

Reply:
I don't know that it "needs to be" I guess.  But I know I haven't defined any urls in Relay matching what crons has done.  They seem be doing some processing Relay.  I'm using the generic forward route.  But if we're exposing an API on sentry we should follow its convention right?

Reply:
https://github.com/getsentry/relay/blob/23fa1f3f358ff9966753dd005ebe94919fa654e7/relay-server/src/endpoints/mod.rs#L108-L109

Reply:
The fallback only cares about that routes need to start with `/api/`.

Reply:
I don't know about Sentry routes, but for me this seems inconsistent, setting the path which is usually referred to as the project_id to 0 just to later still scope it down to a project.

Reply:
Sorry I think I don't have enough context for this.  Do you want to meet up today/tomorrow and teach me more about how Relay handles routes?

The `/0/` is the api version not a project id.

Reply:
> The /0/ is the api version not a project id.

Not for all of [these routes](https://github.com/getsentry/relay/blob/23fa1f3f358ff9966753dd005ebe94919fa654e7/relay-server/src/endpoints/mod.rs#L78-L97). But I am by no means an expert on how Sentry defines its API, I just go off of what I see in Relay ¯\_(ツ)_/¯.

> Sorry I think I don't have enough context for this. Do you want to meet up today/tomorrow and teach me more about how Relay handles routes?

It's relatively simple, it has some handlers defined for a fixed set of routes, everything that doesn't match a specific handler and starts with `/api/` is forwarded to the upstream, everything else is 404'd.

Reply:
Right, I get that but its going to forward to the `sentry.io` host which expects `/api/<version>/`.  So I have to structure my URL in such a way that Relay will forward it with the correct version.  If I do `/api/<project-id>` Sentry will not know how to handle that route.

It works locally with the current route structure so I know its not a functionality problem.  Its more a taste thing how we structure the route.